Doc/library/inspect.rst

```diff
@@ -214,6 +225,17 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 |                   | arguments and local       |
 |                 |                   | variables                 |
 +-----------------+-------------------+---------------------------+
+|                 | co_lines          | returns an iterator that  |
+|                 |                   | yields successive         |
+|                 |                   | bytecode ranges           |
++-----------------+-------------------+---------------------------+
+|                 | co_positions      | returns an iterator of    |
+|                 |                   | source code positions for |
+|                 |                   | each bytecode instruction |
++-----------------+-------------------+---------------------------+
+|                 | replace           | return a copy of the code |
```

Comment:
return -> returns (sorry you'll need to expand the table...)

Reply:
If I understand you correctly, you mean to expand only this section of the table, right? Not the entire table by adding an extra column character (the diff will be an unexpectedly big for this PR..)

Reply:
Either:

- You should align the `|` so you need to expand the entire table (I think?)
- Or you just add a new line before "code" (it'll be a bit ugly, but you won't change the table width at least).